.github/workflows/concrete_compiler_test_cpu_distributed.yml

```diff
@@ -0,0 +1,109 @@
+name: concrete-compiler test linux-cpu-distributed
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - .github/workflows/concrete_compiler_test_cpu_distributed.yml
+      - compilers/concrete-compiler/**
+  push:
+    branches:
+      - 'main'
+      - 'release/*'
+
+env:
+  ACTION_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+  SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL }}
+  SLACK_USERNAME: ${{ secrets.BOT_USERNAME }}
+  SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+
+concurrency:
+  group: concrete_compiler_test_cpu_distributed_${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  setup-instance:
+    runs-on: ubuntu-latest
+    outputs:
+      runner-name: ${{ steps.start-instance.outputs.label }}
+    steps:
+      - name: Start instance
+        id: start-instance
+        uses: zama-ai/slab-github-runner@447a2d0fd2d1a9d647aa0d0723a6e9255372f261
+        with:
+          mode: start
+          github-token: ${{ secrets.SLAB_ACTION_TOKEN }}
+          slab-url: ${{ secrets.SLAB_BASE_URL }}
+          job-secret: ${{ secrets.JOB_SECRET }}
+          backend: aws
+          profile: slurm-cluster
```

Comment:
As discussed IRL, starting already existing EC2 instances (like one of our SLURM cluster) is not yet supporter by Slab. I need to work on this feature. Tell me if it's high priority for the team, I'll make sure to add this feature faster.

Reply:
else yes is high priority for us, to not having both way to start instances. For the transition I will disable this workflow

Reply:
I'll work on it today.